Library/Homebrew/shims/linux/super/make

```diff
@@ -1,26 +1,27 @@
 #!/bin/bash
 
+# shellcheck disable=SC2154,SC2250
```

Comment:
Can you add similar comments above this (and other `disable=`s)?

Reply:
OK, I will add comments like `HOMEBREW_LIBRARY is set by bin/brew` with using ripgrep.